storage/src/vespa/storageframework/generic/clock/time.h

```diff
@@ -45,37 +44,34 @@ uint64_t getRawMicroTime(const Clock&);
  * automatic conversion between time types.
  */
 template<typename Type, int MicrosPerUnit>
-class Time : public boost::operators<Type>
+class Time
```

Comment:
Looks like the majority of member functions here could safely be tagged as `noexcept`

Reply:
That happens further down the PR pipeline.